<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-U2GCSB7P" ignored="0" name="BankingApp Test(failed)(failed)" tests="4" failures="3" timestamp="05 May 2019 11:00:36 GMT" time="38.812" errors="0">
  <testcase name="loginDDT" time="6.405" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="6.18" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="5.826" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginTest" time="2.07" classname="com.inetBanking.testCases.TC_LoginTest_001"/>
</testsuite> <!-- BankingApp Test(failed)(failed) -->
